### PR TITLE
add extrapolation methods

### PR DESCRIPTION
**Pull request by saurabhkgp21**

### CODE REVIEW

File: benchmarks/NonStiffODE/linear_wpd.jmd [hunk 1]
```diff
@@ -213,6 +213,19 @@ wp = WorkPrecisionSet(prob,abstols,reltols,setups;names=solnames,
 plot(wp)
 ```
 
+## Comparison with Extrapolation methods
+
+Now let's test Tsit5 against parallel extrapolation methods:
+```julia
+setups = [Dict(:alg=>Tsit5())
+          Dict(:alg=>AitkenNeville(min_order=1, max_order=9, init_order=9, threading=true))
+          Dict(:alg=>ExtrapolationMidpointDeuflhard(min_order=1, max_order=9, init_order=9, threading=true))]
```
Comment (ChrisRackauckas) [inline benchmarks/NonStiffODE/linear_wpd.jmd]: Why is Hairer-Wanner missing? Isn't that the one that usually does best?

Comment (saurabhkgp21) [reply benchmarks/NonStiffODE/linear_wpd.jmd]: Bug in HairerWanner isn't yet solved. Should I add HairerWanner now or add it after solving that bug?

Comment (ChrisRackauckas) [reply benchmarks/NonStiffODE/linear_wpd.jmd]: wasn't this working before?

Comment (saurabhkgp21) [reply benchmarks/NonStiffODE/linear_wpd.jmd]: HairerWanner will work for this problem, but its not guaranteed to solve any problem we will throw to it. Adding HairerWanner here will work for now, I think. So should I add HairerWanner in benchmarks?

**Pull request merged**